packages/specification/examples/response-bad-request.json

```diff
@@ -0,0 +1,33 @@
+{
+  "resourceType": "Bundle",
+  "type": "transaction-response",
+  "entry": [
+    {
+      "response": {
```

Comment:
Missing fullUrl here

Reply:
Do I insert a new guid here, as there isn't one available when the body fails to parse?